packages/faq-extension/faq.md

```diff
@@ -47,39 +47,39 @@ Notebook. Here are a few of them you may want to try out:
 * Double click on CSV files to view them as a nicely formatted table.
 * Drag and drop notebook cells within a notebook or between notebooks.
 
-See the [JupyterLab Documentation](http://jupyterlab.readthedocs.io/en/latest/)
+See the [JupyterLab Documentation](http://jupyterlab.readthedocs.io/en/stable/)
 for more detailed information about these and other features.
 
 ### How stable is JupyterLab?
 
-This Beta version of JupyterLab is ready for you to use! Starting with this
-release, the JupyterLab Beta series of releases are characterized by:
+This beta version of JupyterLab is ready for you to use! The JupyterLab beta
+series of releases are characterized by:
 
 * Stable and featureful enough for daily usage.
-* Most of the commonly used features in the classic notebook are
+* Most of the commonly used features in the classic Notebook are
   implemented.
-* Developer APIs that are approaching stability but still undergoing
+* Developer APIs that are approaching stability, but still undergoing
   significant changes.
 
-Early in 2018, we will release the 1.0 version of JupyterLab that will
+In 2018, we will release the 1.0 version of JupyterLab that will
```

Comment:
I actually think we don't need to include this paragraph in the FAQ - this I think can stay in the blogpost and docs.